File: BeerViewer/Models/kcsapi/kcsapi_require_info.cs
namespace BeerViewer.Models.kcsapi
{
	public class kcsapi_require_info
	{
		public kcsapi_basic api_basic { get; set; }
		public kcsapi_slotitem[] api_slot_item { get; set; }
		public kcsapi_kdock[] api_kdock { get; set; }
		public kcsapi_useitem[] api_useitem { get; set; }
	}
}

File: WindowFrame/source/System/Interfaces/kcsapi_require_info.ts
import { kcsapi_basic } from "./kcsapi_basic";
import { kcsapi_kdock } from "./kcsapi_dock";
import { kcsapi_slotitem, kcsapi_useitem } from "./kcsapi_item";

export interface kcsapi_require_info {
    api_basic: kcsapi_basic;
    api_slot_item: kcsapi_slotitem[];
    api_kdock: kcsapi_kdock[];
    api_useitem: kcsapi_useitem[];
}